web_src/css/modules/select.css

```diff
@@ -1,25 +0,0 @@
-.gitea-select {
-  position: relative;
-}
-
-.gitea-select select {
-  appearance: none; /* hide default triangle */
-}
-
-/* ::before and ::after pseudo elements don't work on select elements,
-   so we need to put it on the parent. */
-.gitea-select::after {
-  position: absolute;
-  top: 12px;
-  right: 8px;
-  pointer-events: none;
-  content: "";
-  width: 14px;
-  height: 14px;
-  mask-size: cover;
-  -webkit-mask-size: cover;
-  mask-image: var(--octicon-chevron-right);
-  -webkit-mask-image: var(--octicon-chevron-right);
-  transform: rotate(90deg); /* point the chevron down */
-  background: currentcolor;
-}
```

Comment:
@wxiaoguang is this not needed anymore? I recall this class is for "nicer" select boxes with chevron etc, but I think some browser my by default render chevrons as seen in screenshots in https://github.com/go-gitea/gitea/pull/33434, but I'm not sure all of them do.

Reply:
No code uses it.

Reply:
Right, it was a single use. I will check if I need to re-introduce it for https://github.com/go-gitea/gitea/pull/33434.

Reply:
I guess no ....

We already have "ui dropdown (selection)", or just use the browser native "select". I think we'd better avoid too much "customization".

Reply:
Yeah this was a minor enhancement for the native select, because I think those will render differently based on browser OS, but I will check again if it's really needed.

Reply:
Do you mean you need to check before merging this PR, or we could merge this PR if no other problem and leave the "check" to the future?

Reply:
No, merge this now.